requirements.txt

```diff
@@ -1,7 +1,7 @@
 PyGithub
 aiohttp
 discord.py
-audioop-lts # discord.py imports using * syntax 
+# audioop-lts # discord.py imports using * syntax 
```

Comment:
this would break our heroku deployment, so you'd need to also update https://github.com/gpu-mode/discord-cluster-manager/blob/main/runtime.txt

Reply:
I see, let's stick with python 3.11 then.